packages/contracts/test/framework/plugin/plugin-repo.ts

```diff
@@ -152,6 +153,13 @@ describe('PluginRepo', function () {
         expect(toProtocolVersion).to.deep.equal(osxContractsVersion());
       });
     });
+    describe('InitializeFrom', () => {
+      it('Should revert because the function is a placeholder', async () => {
```

Comment:
```suggestion
      it('reverts because the function is a placeholder', async () => {
```
"should" is redundant. I don't know why we write it in other places so often.